CONTRIBUTING.md

```diff
@@ -28,3 +28,32 @@ Members of the Contributors team are encouraged to review pull requests that
 have already been reviewed, and pull request contributors are encouraged to seek
 multiple reviews.  Reviews from anyone not on the Contributors team are always
 appreciated and encouraged!
+
+## Reviewing Pull Requests
+
+All mlpack contributors who choose to review and provide feedback on Pull Requests have a responsibility to both the project and the individual making the contribution. 
+
+Reviews and feedback must be helpful, insightful, and geared towards improving the contribution. If there are reasons why you feel the PR should not be merged, explain what those are. Do not expect to be able to block a Pull Request from advancing simply because you say "No" without giving an explanation. Be open to having your mind changed. Be open to working with the contributor to make the Pull Request better.
+
+Reviews that are dismissive or disrespectful of the contributor or any other reviewers are strictly prohibited.
```

Comment:
In my opinion `strictly prohibited` is a scary word, I might reword this whole sentence positively as

```
Please don't leave dismissive or disrespectful reviews!  It's not helpful for anyone.
```

Reply:
Made apt changes